moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -139,13 +139,11 @@ const DashboardView = () => {
       project["current_status"] = project.project.current_status;
 
       // project status update equivalent to most recent project note
+      // html is parsed before being rendered in the DashboardEditModal component
```

Comment:
Where is it being parsed?

Reply:
it is sent into the DashboardEditModal component [here](https://github.com/cityofaustin/atd-moped/blob/ae264a5d4ba55e8639d0fd48c4f92564f6ac42db/moped-editor/src/views/dashboard/DashboardView.js#L211) and parsed [here](https://github.com/cityofaustin/atd-moped/blob/ae264a5d4ba55e8639d0fd48c4f92564f6ac42db/moped-editor/src/views/dashboard/DashboardEditModal.js#L48)